gulp-tasks/build.js

```diff
@@ -20,6 +20,10 @@ gulp.task('build:announcement', function() {
   const globOpts = {
     srcBase: 'src/content/en/',
     prefixBase: true,
+    ignore: [
```

Comment:
Why are you disabling this? It won't be disabled on the real site.

Reply:
Because I don't want the announcements on workbox.

Reply:
Understood, but we want to be able to put them across all pages, so for now, let's leave them on and we can talk about removing them later.